lib/src/jpegr.cpp

```diff
@@ -726,7 +726,7 @@ status_t JpegR::decodeJPEGR(jr_compressed_ptr jpegr_image_ptr, jr_uncompressed_p
     if (gainmap_image_ptr != nullptr) {
       gainmap_image_ptr->pixelFormat = gainmap_image.pixelFormat;
       if (jpeg_dec_obj_yuv420.getDecompressedImageWidth() % gainmap_image.width == 0 &&
-          jpeg_dec_obj_yuv420.getDecompressedImageHeight() % gainmap_image.width == 0) {
```

Comment:
perhaps this commit is not required. 
It can squashed with parent change and force pushed instead